### PR TITLE
Add graphite test + rewrite graphite object + docu

### PR DESCRIPTION
**Pull request by Reamer**
Hi,

some little changes for graphite object, changes in docu after last merge and tests for graphite

Greetings Reamer

### CODE REVIEW

File: manifests/object/graphitewriter.pp [hunk 1]
```diff
@@ -10,48 +10,25 @@
 #
 
 define icinga2::object::graphitewriter (
-  $graphite_host        = '127.0.0.1',
-  $graphite_port        = 2003,
+  $host       = '127.0.0.1',
+  $port       = 2003,
```
Comment (arioch) [inline manifests/object/graphitewriter.pp:14]: I fear $host and $port are or may to end up on puppet's reserved keywords list.
So it's safer to keep the prefix.

**Comment by Reamer** [comment]
@arioch In PR https://github.com/Icinga/puppet-icinga2/pull/146 @lazyfrosch wrote "avoid the gelf_prefix"
So whats the best way? I don't think that $host and $port may to end up on puppet's reserved keywords list.
$port is used in a lot of other module (example: https://github.com/puppetlabs/puppetlabs-apache/blob/master/manifests/vhost.pp)
$host is also used in popular other module (example: https://github.com/puppetlabs/puppetlabs-mysql/blob/master/manifests/db.pp)

In my own modules i also avoid prefix, because is a double in hiera icinga2::object::**graphite**writer::**graphite**_host

Greetings Reamer

**Comment by lazyfrosch** [comment]
I'm with dropping that prefix. We can't predict everything.

And @Reamer's examples proves a good practice, even from Puppetlabs.

**Comment by arioch** [comment]
Alright, guess you guys have a point.
Merged and thanks again, @Reamer keep those patches coming. :+1:

**Pull request closed without merging**